node/src/bin/manager.rs

```diff
@@ -179,6 +179,13 @@ pub enum Command {
     Chain(ChainCommand),
     /// Manipulate internal subgraph statistics
     Stats(StatsCommand),
+    /// Perform a SQL ANALYZE in a Entity table
+    Analyze {
```

Comment:
Perhaps the command could accept a `verbose` argument?
I would even argue we want verbose as default.

Reply:
I agree .. it might be a little tricky to get the command output out of diesel, but if it's not too much work, would be nice to just dump whatever Postgres says onto the tty.

Reply:
I think this command should be `graphman stats analyze Qm.. Entity`, i.e., the `analyze` should be a subcommand of `stats`

Reply:
I couldn't find a way to capture PostgreSQL output from diesel. Not sure if this is supported at all.